ckanext/data_qld/fanstatic/flag-comment.js

```diff
@@ -46,6 +46,7 @@ this.ckan.module('flag-comment', function (jQuery) {
      * Returns nothing.
      */
     initialize: function () {
+      jQuery('#comment_form').append("DEBUG: flag-comment.js:initialize");
```

Comment:
do you still want debug in this repo?

Reply:
I'm still encountering some issues with the JavaScript on the flagging test, just for CKAN 2.9, so I'll keep it for now, but I should delete this before merging.